google/ads/google_ads/config.py

```diff
@@ -129,6 +129,25 @@ def load_from_yaml_file(path=None):
 
     return parse_yaml_document_to_dict(yaml_doc)
 
+@_config_validation_decorator
+@_config_parser_decorator
+def load_from_dict(config_dict):
+    """ Check if the argument is dictionary or not
+
+    Args:
+        Configuration dictonary of credentials
+
+    Returns:
+        The same input dictionary
+
+    Raises:
+        ValueError if it is not a dictionary
+    """
```

Comment:
In all of the above doc comments try to use less ambiguous language and be sure to include punctuation. 

```
The same dict that is passed into the function.
...
A ValueError if the input is not a dict.
...
```